addons/copy-message-link/profile.js

```diff
@@ -1,4 +1,8 @@
 export default async function ({ addon, console, msg }) {
+  addon.self.addEventListener("disabled", () => {
+    document.querySelectorAll(".sa-copy-link-btn").forEach(element => element.remove());
+  })
+  
```

Comment:
This is not enough
You need to make sure the "copy link" button is no longer added if the addon is disabled (boolean `addon.self.disabled`) and also check that reenabling the addon adds the buttons back.

Reply:
> You need to make sure the "copy link" button is no longer added if the addon is disabled

What do you mean? The buttons are deleted when it's disabled and they don't come back.

> and also check that reenabling the addon adds the buttons back

I'll fix that real quick

Reply:
> and they don't come back.

Have you tried loading more pages of comments after disabling the addon?

Reply:
Just tried, works perfectly